src/plugins.js

```diff
@@ -0,0 +1,131 @@
+export default {
+  // es2015
+  "transform-es2015-arrow-functions": {
+    edge: 13,
+    firefox: 45,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-block-scoped-functions": {
+    edge: 11,
+    firefox: 46,
+    chrome: 41,
+    opera: 28,
+  },
+  "transform-es2015-block-scoping": {
+    edge: 11,
+    firefox: 36,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-classes": {
```

Comment:
This needs to be enabled wherever super and spread are not both available at the same time.

This is easier to trigger when using `transform-class-properties`, but `class X extends Y { constructor (...args) { super(...args) } }` cannot be transformed by spread, as it's not possible to call `super` as a Function object.

Going by the version numbers you specified, this seems to be ok with browsers, but, again, this is a problem in Node 4.